server/src/test/java/org/elasticsearch/index/query/RangeQueryBuilderTests.java

```diff
@@ -244,11 +268,7 @@ public void testIllegalArguments() {
     }
 
     public void testToQueryNumericField() throws IOException {
-        Query parsedQuery = rangeQuery(INT_FIELD_NAME).from(23)
-            .to(54)
-            .includeLower(true)
-            .includeUpper(false)
-            .toQuery(createSearchExecutionContext());
+        Query parsedQuery = rangeQuery(INT_FIELD_NAME).gte(23).lt(54).toQuery(createSearchExecutionContext());
```

Comment:
I had to update these tests to use the new notation. They started failing because now if the old notation is used, we can't always guarantee that the roundtrip query builder -> JSON -> query builder produces the exact same builder. For example, the builder could have `from: null, include_lower: false`, and this would roundtrip back to a builder with `from: null, include_lower: true`.